src/Codec/Xlsx/Types/Common.hs

```diff
@@ -385,3 +411,76 @@ instance ToAttrVal ErrorType where
   toAttrVal ErrorNum = "#NUM!"
   toAttrVal ErrorRef = "#REF!"
   toAttrVal ErrorValue = "#VALUE!"
+
+type RowIndex = Int
```

Comment:
I added the alias was because I sometimes got confused between columns and row indexes.
The documentation already makes a distinction between rowIndex and columnIndex in many places, so I thought a logical step further would be an alias.

Changing this to a newtype would be better because mixing up of column/rows becomes less likely.
Integers for RowIndex and ColumnIndex are ubiquitous in the xlsx code base.
So I think replacing it with a newtype isn't a good idea for this PR because it'd mean rewriting a large part of the xlsx API surface.

Actually, a newtype would be really good because it allows us to define smart constructors which disallow indexes < 1. We've had a production bug on this. Excel will warn users that it's being hacked if that happens.

Should I delete these aliases for now?

Reply:
Probably it's better to have it in a separate PR just to limit the scope of this one, my only concern would be not to forget about this :)